infra/ansible/development.Dockerfile

```diff
@@ -17,3 +17,8 @@ ENV TAGS="bazel,configure_env,install_deps"
 ARG ansible_vars
 RUN ansible-playbook playbook.yaml -e "stage=build" -e "${ansible_vars}" --tags "${TAGS}"
 RUN ansible-playbook playbook.yaml -e "stage=release" -e "${ansible_vars}" --tags "${TAGS}"
+
+RUN useradd --create-home --shell /bin/bash ptxla
```

Comment:
Can you move this to a separate PR? This is much riskier than updating the Python version.

Reply:
I'll see if there's a way to use the TPU as a non-root user in a container in the meantime.

Reply:
ah, +1 https://github.com/pytorch/xla/pull/6685/files#r1516655463

Reply:
Will remove that.